pkg/controller/compliancescan/compliancescan_controller.go

```diff
@@ -573,8 +573,8 @@ func (r *ReconcileComplianceScan) generateResultEventForScan(scan *compv1alpha1.
 
 	if scan.Status.Result == compv1alpha1.ResultNotApplicable {
 		r.recorder.Eventf(
-			scan, corev1.EventTypeNormal, "ScanNotApplicable",
-			"The scan result is not applicable, please check if you're using the correct platform")
+			scan, corev1.EventTypeWarning, "ScanNotApplicable",
+			"The scan result is not applicable, please check if you're using the correct platform or if the nodeSelector matches nodes.")
```

Comment:
Should we suggest nodes or pool?

Reply:
I would add `nodes` here given that it's canonically what should happen... the fact that the nodeSelector should ideally matcha pool should still be indicated as part of the documentation. But it's not strictly necessary, it just won't generate remediations.